setup.py

```diff
@@ -2,12 +2,13 @@
 import os
 
 from setuptools import Extension, find_packages, setup
+from Cython.Build import cythonize
```

Comment:
We didn't use Cython as dependency. Why did you decide to add it?

Reply:
I took your setup.py from the dropout branch for some reason. You had added an import of cython to it